presto-docs/src/main/sphinx/connector/thrift.rst

```diff
@@ -0,0 +1,96 @@
+================
+Thrift Connector
+================
+
+The Thrift connector makes it possible to integrate with external storage systems
+without a custom Presto connector implementation.
+
+In order to use the Thrift connector with external system, you need to implement
+the ``PrestoThriftService`` interface, found below. Next, you configure the Thrift connector
+to point to a set of machines, called Thrift servers, that implement the interface.
+As part of the interface implementation, the Thrift servers will provide metadata,
+splits and data. The Thrift server instances are assumed to be stateless and independent
+from each other.
+
+Configuration
+-------------
+
+To configure the Thrift connector, create a catalog properties file
+``etc/catalog/thrift.properties`` with the following content,
+replacing the properties as appropriate:
+
+.. code-block:: none
+
+    connector.name=presto-thrift
+    static-location.hosts=host:port,host:port
+
+Multiple Thrift Systems
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can have as many catalogs as you need, so if you have additional
+Thrift systems to connect to, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``).
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+===========================================   ==============================================================
+Property Name                                 Description
+===========================================   ==============================================================
+``static-location.hosts``                     Location of Thrift servers
+``presto-thrift.max-response-size``           Maximum size of a response from thrift server
+``presto-thrift.metadata-refresh-threads``    Number of refresh threads for metadata cache
+===========================================   ==============================================================
+
+``static-location.hosts``
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Comma-separated list of thrift servers in the form of ``host:port``. For example:
+
+.. code-block:: none
+
+    static-location.hosts=localhost:7777,localhost:7779
```

Comment:
Let's use fixed IP addresses from RFC 5737 as the example. I don't think "localhost" would be valid except for a single-node Presto cluster. How about
```
static-location.hosts=192.0.2.3:7777,192.0.2.4:8888